src/lua/error.lua

```diff
@@ -168,11 +180,12 @@ local function error_serialize(err)
 end
 
 local error_methods = {
-    ["unpack"] = error_unpack;
-    ["raise"] = error_raise;
-    ["match"] = error_match; -- Tarantool 1.6 backward compatibility
-    ["__serialize"] = error_serialize;
-    ["set_prev"] = error_set_prev;
+    ["unpack"]       = error_unpack;
+    ["raise"]        = error_raise;
+    ["copy"]         = error_copy;
+    ["match"]        = error_match; -- Tarantool 1.6 backward compatibility
+    ["__serialize"]  = error_serialize;
+    ["set_prev"]     = error_set_prev;
```

Comment:
Please avoid unrelated changes. There's no need to align this table now. Let's leave it as is and follow the style.

```diff
     ["__serialize"] = error_serialize;
+    ["set_prev"] = error_set_prev;
 }
```